### PR TITLE
doc(README.md): compilation instructions

### PR DESCRIPTION
**Pull request by jbrouze**
# Pull Request Template

## Description

Please include a short resume of the changes and what is the purpose of PR. Any relevant information should be added to help:
* **QA Team** (Quality Assurance) with tests.
* **reviewers** to understand what are the stakes of the pull request.

**Fixes** # (issue)

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [x] Updating documentation (missing information, typo...)

## Target serie

- [ ] 18.10.x
- [ ] 19.04.x
- [ ] 19.10.x
- [x] 20.04.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [ ] I followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

#### Centreon team only

- [ ] I have made sure that the **unit tests** related to the story are successful.
- [ ] I have made sure that **unit tests cover 80%** of the code written for the story.
- [ ] I have made sure that **acceptance tests** related to the story are successful (**local and CI**)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,8 +69,27 @@ This paragraph is only a quickstart guide for the compilation of
 Centreon Engine. For a more in-depth guide with build options you should
 refer to the [online documentation](https://documentation.centreon.com/docs/centreon-engine/en/latest/installation/index.html#using-sources).
 
-Centreon Engine needs Centreon Clib to be build. You should
-[install it first](https://github.com/centreon/centreon-clib).
+
+"""
+()[For Centos 7]() :
```
Comment (bouda1) [inline README.md:74]: Syntax to update here...

File: README.md [hunk 2]
```diff
@@ -65,8 +69,27 @@ This paragraph is only a quickstart guide for the compilation of
 Centreon Engine. For a more in-depth guide with build options you should
 refer to the [online documentation](https://documentation.centreon.com/docs/centreon-engine/en/latest/installation/index.html#using-sources).
 
-Centreon Engine needs Centreon Clib to be build. You should
-[install it first](https://github.com/centreon/centreon-clib).
+
+"""
```
Comment (bouda1) [inline README.md:73]: Bad syntax

File: README.md [hunk 3]
```diff
@@ -65,8 +69,27 @@ This paragraph is only a quickstart guide for the compilation of
...
+()[For Centos 7]() :
+First of all, check if you have these packages installed (Note that packages names come from Centos 7 distribution, so if some packages names don't match on your distribution try to find their equivalent names) :
+
+    git, make, cmake.
```
Comment (bouda1) [inline README.md:77]: gcc-c++, python3, epel-release, cmake3

File: README.md [hunk 4]
```diff
@@ -65,8 +69,27 @@ This paragraph is only a quickstart guide for the compilation of
...
+
+If they are not installed, pleade install them.
```
Comment (bouda1) [inline README.md:79]: pleade ??

Comment (bouda1) [reply README.md:79]: You have to specify how to install conan.
For centos7, do prefer pip3 install conan
For other OS, do prefer the one given by the OS package manager

File: README.md [hunk 5]
```diff
@@ -65,8 +69,27 @@ This paragraph is only a quickstart guide for the compilation of
...
+
+    $> git clone https://github.com/centreon/centreon-engine
+    $> cd centreon-engine && ./make.sh
+    $> cd build
+    $> make && make install
+
+
+()[For other didtributions]() :
```
Comment (bouda1) [inline README.md:87]: () ???

File: README.md [hunk 6]
```diff
@@ -65,8 +69,27 @@ This paragraph is only a quickstart guide for the compilation of
...
+If you are on another distribution, then follow the steps below.
+
+For the projet compilation you need to have conan installed. To install conan you need pip3 if you are using python3 (python package manager). You can install conan in that manner.
```
Comment (bouda1) [inline README.md:90]: Explain how to install it. Prefer the version given by the OS if available.

Comment (bouda1) [reply README.md:90]: Also specify that conan should be installed after gcc

File: README.md [hunk 7]
```diff
@@ -65,8 +69,27 @@ This paragraph is only a quickstart guide for the compilation of
...
+
+    $> pip3 install conan
 
 Once the sources of Centreon Engine extracted go to the *./build/*
```
Comment (bouda1) [inline README.md:94]: It does not exist, we have to create it...

File: README.md [hunk 8]
```diff
@@ -75,7 +98,9 @@ everything went fine.
 
     $> cd centreon-engine/build
```
Comment (bouda1) [inline README.md:99]: miss the git command to clone

File: README.md [hunk 9]
```diff
@@ -75,7 +98,9 @@ everything went fine.
...
     $> cmake .
```
Comment (bouda1) [inline README.md:100]: remove this line.

File: README.md [hunk 10]
```diff
@@ -75,7 +98,9 @@ everything went fine.
...
-    ...
+    $> cmake" -DWITH_PREFIX=/usr -DWITH_PREFIX_BIN=/usr/sbin -DWITH_USER=centreon-engine -DWITH_GROUP=centreon-engine -DCMAKE_BUILD_TYPE=Debug -DWITH_RW_DIR=/var/lib/centreon-engine/rw -DWITH_PREFIX_CONF=/etc/centreon-engine -DWITH_VAR_DIR=/var/log/centreon-engine -DWITH_PREFIX_LIB=/usr/lib64/centreon-engine -DWITH_TESTING=On -DWITH_SIMU=On -DWITH_CREATE_FILES=OFF -DWITH_BENCH=On ..
```
Comment (bouda1) [inline README.md:101]: Remove "

Comment (bouda1) [inline README.md:101]: -DCMAKE_BUILD_TYPE=Release

Comment (bouda1) [reply README.md:101]: -DWITH_SIMU=On is not needed for an end user. Also, -DWITH_BENCH= On is not needed

File: README.md [hunk 11]
```diff
@@ -75,7 +98,9 @@ everything went fine.
...
+
+"""
```
Comment (bouda1) [inline README.md:103]: Read the markdown tutorial.

File: README.md [hunk 12]
```diff
@@ -84,8 +109,12 @@ software by running *make install* as priviledged user.
     ...
     $# make install
```
Comment (bouda1) [inline README.md:111]: You can make something shorter

File: README.md [hunk 13]
```diff
@@ -84,8 +109,12 @@ software by running *make install* as priviledged user.
...
 
+    or : 
+    $> make && make install
+
 You're done !
 
+
```
Comment (bouda1) [inline README.md:117]: Verify /etc/centreon-engine directory.

Usually, its content is filled by the Centreon web interface, owned by the apache user.

A command like the following may be necessary:

chown -R apache:apache /etc/centreon-engine/*

File: README.md [hunk 14]
```diff
@@ -65,27 +69,63 @@ This paragraph is only a quickstart guide for the compilation of
...
-everything went fine.
+Note that conan should be installed after gcc-c++
 
+Exemple :
+    $> _apt_  install conan
+
+If it doesn't work, you will have to install a python programm such as pip3 :
+    $> _apt_ install pip3
+Then :
+    $> pip3 install conan
```
Comment (bouda1) [inline README.md:103]: If it does not work, conan can be installed with :
$> pip3 install conan

Comment (bouda1) [reply README.md:103]: Again, for other OS you need to specify what is needed to compile centreon-engine:
cmake, gcc-g++, make, etc...

File: README.md [hunk 15]
```diff
@@ -65,27 +69,64 @@ This paragraph is only a quickstart guide for the compilation of
...
+    $> cd build
+    $> make && make install
+
+
+**For other didtributions :**
+If you are on another distribution, then follow the steps below.
+
+Check if you have these packages installed (Note that packages names come from Centos 7 distribution, so if some packages names don't match on your distribution try to find their equivalent names) :
+
+    git, make, cmake, gcc-c++, python3, cmake3
```
Comment (SylvestreG) [inline README.md:95]: cmake is not necessary if you have cmake3

File: README.md [hunk 16]
```diff
@@ -65,27 +69,64 @@ This paragraph is only a quickstart guide for the compilation of
...
 
+Exemple :
+    $> _apt_  install conan
+
+If it does not work, conan can be installed with pip3 :
+    $> pip3 install conan
+
+You can now compile centreon-engine :
+    $> git clone https://github.com/centreon/centreon-engine
     $> cd centreon-engine/build
```
Comment (SylvestreG) [inline README.md:107]: the build directory is not here by default you can replace 
$> cd centreon-engine/build
by
$> mkdir -p centreon-engine/build
$> cd centreon-engine/build

**Pull request merged**